internal/jujuapi/streamproxy_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 Canonical.
+
+package jujuapi_test
+
+import (
+	"context"
+
+	"github.com/juju/juju/api"
+	"github.com/juju/juju/api/common"
+	gc "gopkg.in/check.v1"
+
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/openfga"
+	ofganames "github.com/canonical/jimm/v3/internal/openfga/names"
+)
+
+type streamProxySuite struct {
+	websocketSuite
+}
+
+var _ = gc.Suite(&streamProxySuite{})
+
+func (s *streamProxySuite) TestDebugLogs(c *gc.C) {
```

Comment:
i suggest to expand the debug log good test with some actual debug logs, if that's possible

Reply:
For some reason the Juju controller deployed as part of the conn suite doesn't return debug logs. Maybe because there is nothing in the model.

Reply:
Will leave this for the follow up as discussed.